cartridges/app_yd_core/cartridge/controllers/Account.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+var server = require('server');
+server.extend(module.superModule);
+
+/**
+ * Account-SaveNewPassword : The Account-SaveNewPassword endpoint handles resetting a shoppers password. This is the last step in the forgot password user flow. (This step does not log the shopper in.)
+ * @name Base/Account-SaveNewPassword
+ * @function
+ * @memberof Account
+ * @param {middleware} - server.middleware.https
+ * @param {httpparameter} - token - SFRA utilizes this token to retrieve the shopper
+ * @param {httpparameter} - dwfrm_newPasswords_newpassword - Input field for the shopper's new password
+ * @param {httpparameter} - dwfrm_newPasswords_newpasswordconfirm  - Input field to confirm the shopper's new password
+ * @param {httpparameter} - save - unutilized param
+ * @param {category} - sensitive
+ * @param {renders} - isml
+ * @param {serverfunction} - post
+ */
+server.replace('SaveNewPassword', server.middleware.https, function (req, res, next) {
```

Comment:
we need to restrict access to this endpoint for GET requests:
```suggestion
server.replace('SaveNewPassword', server.middleware.post, server.middleware.https, function (req, res, next) {
```

Reply:
updated